content/components/recipes/form/arguments/MultiPreferenceBranches.js

```diff
@@ -230,6 +231,7 @@ function Branch({ index }) {
     return parseInt(value, 10);
   };
 
+  const errorPrefix = `arguments.branches.${index}.`;
```

Comment:
There is also room to have errors specifically on `arguments.branches`, for example if no branches are added. I think we should have an error display for that as well.